exercises/practice/rna-transcription/.meta/example.awk

```diff
@@ -0,0 +1,21 @@
+BEGIN {
+    FS = ""
+    translation["G"] = "C"
+    translation["C"] = "G"
+    translation["A"] = "U"
+    translation["T"] = "A"
+
+}
+
+{
+    out = ""
+    for (i = 1; i <= NF; i++) {
+        if (translation[$i]) {
```

Comment:
In general, better to write `if ($i in translation)`
That explicitly tests the array has the key.
For a non-existing key, `translation[$i]` actually adds the key to the array, and `translation[$i]` has type "unassigned".

It does produce the right outcome though.

Reply:
```suggestion
        if ($i in translation) {
```

Reply:
Thanks!